public/js/components/AtomEdit/CustomEditors/GuideEditor.js

```diff
@@ -28,6 +29,11 @@ export class GuideEditor extends React.Component {
               <GuideItem onFormErrorsUpdate={this.props.onFormErrorsUpdate} />
             </FormFieldArrayWrapper>
           </ManagedField>
+          <ManagedField fieldLocation="commissioningDesks" name="Commissioning desks" updateData={this.onDesk}>
```

Comment:
Where is `onDesk` coming from?

Reply:
ha! good catch, I removed it but forgot to remove the attribute too.